packages/scaffold/src/views/w3m-connecting-social-view/index.ts

```diff
@@ -24,10 +26,14 @@ export class W3mConnectingSocialView extends LitElement {
   // -- State & Properties -------------------------------- //
   @state() private socialProvider = AccountController.state.socialProvider
 
+  @state() private socialWindow = AccountController.state.socialWindow
+
   @state() protected error = false
```

Comment:
Where these `error` and `connecting` are got updated? Cannot see they are updated with a listener or something. 

I'm concerned if we should call `this.getMessage()` in every render by calling it on `render()` funciton. 

If we are listening those values, we'd better to update this.message in the event listener or in life cycle methods IMO.